editor/src/frontend/utility_types.rs

```diff
@@ -6,3 +6,12 @@ pub struct FrontendDocumentDetails {
 	pub name: String,
 	pub id: u64,
 }
+
+#[derive(Clone, Copy, Debug, Eq, Deserialize, PartialEq, Serialize)]
+pub enum FrontendMouseCursor {
```

Comment:
"Frontend" is perhaps redundant here. Maybe `MouseCursorIcon` would be more descriptive.